index.html

```diff
@@ -67,122 +85,134 @@ <h1>Hey there. I’m Mamun</h1>
-                        <li><a href="#">JavaScript</a></li>
-                        <li><a href="#">html</a></li>
-                    </ul>
+            </article>
+            <article class="c-posts__item card-wrapper">
+                <div class="card-image-wrapper"></div>
+                <div class="card-content-wrapper">
+                    <h2 class="work-card-title">Multi-Post Stories Gain + Glorys</h2>
+                    <div class="tech-tags">
+                        <ul>
+                            <li><a href="#">Ruby on rails</a></li>
+                            <li><a href="#">css</a></li>
+                            <li><a href="#">JavaScript</a></li>
+                            <li><a href="#">html</a></li>
+                        </ul>
+                    </div>
+                    <button type="button" class="c-btn">See Project</button>
                 </div>
-                <button type="button" class="c-btn">See Project</button>
-            </div>
-        </article>
+            </article>
+        </div>
     </section>
 
     <section id="about-me" class="c-posts">
-        <article class="c-posts__item">
-            <h2 class="section-title mb-0">About me</h2>
-            <p class="about-text">
-                I can help you build a product , feature or website Look through some of my work and experience! If you
-                like what you see and have a project you need coded,
-                don’t hestiate to contact me.
-            </p>
-            <button type="button" class="c-btn">Get My Resume</button>
-        </article>
-        <article class="c-posts__item card-wrapper skill-main-wrapper">
-            <div class="skill-image-wrapper">
-                <div class="">
+        <div class="container">
+            <article id="about" class="c-posts__item">
+                <div class="left-bg-img">
+                    <img src="assets/img/about-left.svg">
+                </div>
+                <div class="about-content">
+                    <h2 class="section-title mb-0">About me</h2>
+                    <p class="about-text">
+                        I can help you build a product , feature or website Look through some of my work and experience!
+                        If
+                        you
+                        like what you see and have a project you need coded,
+                        don’t hestiate to contact me.
+                    </p>
+                    <button type="button" class="c-btn">Get My Resume</button>
+                </div>
+                <div class="right-bg-img">
+                    <img src="assets/img/about-right.svg">
+                </div>
+            </article>
+        </div>
+        <div class="items">
+            <article class="c-posts__item card-wrapper skill-main-wrapper">
+                <div class="skill-image-wrapper">
                     <img src="assets/img/icon-languages.svg"/>
                 </div>
```

Comment:
- [ ] [Optional] I would suggest you please stick to the design as much as possible. please adjust the spacing and also every card on `About me` has the same width please fix it
### Your design
![his](https://user-images.githubusercontent.com/81727988/214857803-ceb757d0-7533-4373-8d31-d4095c4b2ede.jpg)

### Figma design

![fimga](https://user-images.githubusercontent.com/81727988/214857851-36369530-b15e-4639-ab10-09f39cd2b7b9.jpg)